src/server/components/lists/helpers.ts

```diff
@@ -233,14 +233,18 @@ export function createListSearchBaseLink(serviceType: string): string {
   return `${protocol}://${SERVICE_DOMAIN}${listsRoutes.finder}?serviceType=${serviceType}`;
 }
 
-export function createFormRunnerReturningUserLink(serviceType: string): string {
+export function createFormRunnerReturningUserLink(serviceType: string, isAnnualReview: boolean): string {
+  const formName = kebabCase(serviceType);
```

Comment:
shouldn't formName be declared after the guard conditions?

Reply:
yep